content/en/Getting started/what-to-expect.md

```diff
@@ -40,17 +40,19 @@ results. Here's what you can expect:
 1. Once the pentest is complete, we move your pentest from _Planned_ to _Remediation_.
 1. You can start assessing all discovered vulnerabilities. In the Cobalt app, navigate
    to **Pentests**. Select your pentest, and navigate to the **Findings** tab.
-   - Scroll down until you see **Activity**. Depending on your assessment, you can
+   - Find the **Status** list on the right of the page. Depending on your assessment, you can
```

Comment:
I'm not sure that we need to locate the **Status** list (maybe we'll move it in the future), and "Command-F" works in our UI. And the [Google Developer Style Guide](https://developers.google.com/style/ui-elements) discourages it

```suggestion
   - Find the **Status** list. Depending on your assessment, you can
```

<img width="1341" alt="Screen Shot 2022-10-05 at 10 14 39 AM" src="https://user-images.githubusercontent.com/84352037/194121380-c0c8b282-d4d9-4091-a08f-a8e4364c91be.png">

Reply:
@mjang-cobalt I added the location of the list because we're moving it from the bottom of the page to the panel on the right with this release. But you're right, probably it's not needed.

Reply:
We don't have to follow style guides 100% of the time. But in this case, I think it makes sense.

Reply:
Want to make sure we update this to State vs. Status. We have the term "State" in the UI vs. Status (will be updating that)